content/en/docs/reference/config-api/kubelet-config.v1beta1.md

```diff
@@ -1637,6 +1637,13 @@ option is explicitly enabled.
 Default: false</p>
 </td>
 </tr>
```

Comment:
Don't change this file yet. As mentioned in the head of the file, it is auto-generated.
Your change will be overwritten anyway when the next release is out.

Reply:
fixed